vpr/CMakeLists.txt

```diff
@@ -64,7 +64,9 @@ set(VPR_COMPILE_OPTIONS_FLAGS
     "-Werror"
     )
 
-if (VPR_COMPILE_OPTIONS STREQUAL "strict")
+set(CXX_VERSION $ENV{CXX})
+
+if (VPR_COMPILE_OPTIONS STREQUAL "strict" AND CXX_VERSION STREQUAL "g++-6")
```

Comment:
I think it would be better to handle this in the Travis configuration rather than in CMake.

That way the VPR_COMPILE_OPTIONS option is just a simple configuration setting. As written It would seem rather mysterious if a developer set a strict build but wasn't using g++-6 and didn't error out on warnings.

Reply:
Makes sense, fixed. I have added the VPR_FLAGS env in `travis.yml` to the main VtR build and the current compatibility builds that do not produce warnings.